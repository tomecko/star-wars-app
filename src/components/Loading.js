import React from 'react';
import { Card, CardBody, Spinner } from 'reactstrap';

export function Loading() {
  return (
    <Card>
      <CardBody>
        <Spinner type="grow" size="sm" color="primary" />{' '}
        Loadingâ€¦
      </CardBody>
    </Card>
  );
}

