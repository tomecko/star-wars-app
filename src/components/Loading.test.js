import React from 'react';
import renderer from 'react-test-renderer';

import { Loading } from './Loading';

test('matches snapshot', () => {
  const component = renderer.create(
    <Loading />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
