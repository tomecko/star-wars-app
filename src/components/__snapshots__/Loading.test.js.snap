// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card-body"
  >
    <div
      className="spinner-grow-sm spinner-grow text-primary"
      role="status"
    >
      <span
        className="sr-only"
      >
        Loading...
      </span>
    </div>
     
    Loadingâ€¦
  </div>
</div>
`;
